src/runtime_src/core/common/api/xrt_module.cpp

```diff
@@ -38,7 +38,7 @@ namespace
 // 0 if no padding is required.   The page size should be
 // embedded as ELF metadata in the future.
 static constexpr size_t column_page_size = AIE_COLUMN_PAGE_SIZE;
-static constexpr uint8_t Elf_Amd_Aie2p  = 66;
+static constexpr uint8_t Elf_Amd_Aie2p  = 69;
 static constexpr uint8_t Elf_Amd_Aie2ps = 64;
```

Comment:
What do these constants represent?  
My preference would be all lower case, e.g. `elf_amd_aie2p`, but I don't care enough.

Reply:
Please refer to this confluence page for more information:
https://confluence.amd.com/display/AIE/IPU+Execution+Buffer+ELF+Specification
A screen shot of the confluence page is shown as below:
<img width="1006" alt="image" src="https://github.com/Xilinx/XRT/assets/148396430/5aac6f56-b8c9-4a7c-9e0b-1d8404b5f6a1">